layouts/shortcodes/pdogpay.html

```diff
@@ -0,0 +1,16 @@
+<div id="list-example" class="list-group">
+  <a class="list-group-item list-group-item-action" href="#list-item-1">Item 1</a>
+  <a class="list-group-item list-group-item-action" href="#list-item-2">Item 2</a>
+  <a class="list-group-item list-group-item-action" href="#list-item-3">Item 3</a>
+  <a class="list-group-item list-group-item-action" href="#list-item-4">Item 4</a>
+</div>
+<div data-spy="scroll" data-target="#list-example" data-offset="0" class="scrollspy-example">
+  <h4 id="list-item-1">Item 1</h4>
+  <p>...</p>
+  <h4 id="list-item-2">Item 2</h4>
+  <p>...</p>
+  <h4 id="list-item-3">Item 3</h4>
+  <p>...</p>
+  <h4 id="list-item-4">Item 4</h4>
+  <p>...</p>
+</div>
```

Comment:
Dette ser ut som det ikke er helt klart for å sjekkes inn i repoet ennå

Reply:
Den er lagt inn på mal, hvor jeg har litt diverse tester. Hvis du vil, kan du se på filen og se hvorfor den ikke vises iht. Bootstrap-dokumentasjonen?

Reply:
Disse elementene vises for meg når jeg kjører `hugo serve` og besøker <http://localhost:1313/mal/> i nettleseren. Hva mener du ikke vises?